specification.markdown

```diff
@@ -778,3 +778,14 @@ minOccurs: 0
 type: Binary
 
 description: Used to void damaged data, to avoid unexpected behaviors when using damaged data. The content is discarded. Also used to reserve space in a sub-element for later use.
+
+# Considerations for Reading EBML Data
+
+The following scenarios describe events when reading `EBML Documents` and the REQUIRED behavior by an `EBML Reader`.
+
+If a `Master Element` contains a `CRC-32 Element` that doesn't validate, then the `EBML Reader` MUST ignore all contained data except for `Descendant Elements` which contain their own valid `CRC-32 Element`.
+
+If a `Master Element` contains more occurrences of a `Child Master Element` than permitted according to the `maxOccurs` and `recurring` attributes of the definition of that `Element` then the occurrences in addition to `maxOccurs` MUST be ignored.
```

Comment:
`SHOULD` ? Although it would create different behaviors depending on the internal limit they set.